data/bunrodea/bunrodea weapons.txt

```diff
@@ -50,7 +50,7 @@ outfit "Locust Turret"
 		sound "locust blaster"
 		"hit effect" "bullet impact"
 		"inaccuracy" 5
-		"turret turn" 6
+		"turret turn" 2.
```

Comment:
I wouldnt add a "." if it isnt required, ig this probably just is the remainder of a 2.1 speed or smth

Reply:
I noticed that the convention seemed to be to use a ".", at least, there are a few other existing turrets that do, so I was keeping the formatting consistent. 
I can remove it though if we don't want to do that.

See:
https://github.com/endless-sky/endless-sky/blob/ea07dbe701c2061c6efbed342eaf23f76a69b4d5/data/quarg/quarg%20outfits.txt#L53

https://github.com/endless-sky/endless-sky/blob/ea07dbe701c2061c6efbed342eaf23f76a69b4d5/data/drak/drak%20outfits.txt#L25

https://github.com/endless-sky/endless-sky/blob/ea07dbe701c2061c6efbed342eaf23f76a69b4d5/data/drak/drak%20outfits.txt#L158

Reply:
would be pretty weird to have it just for turret turn rates and not for all other numbers, but fair enough

Reply:
More integer turret turn rate values don't have the trailing decimal point than do.
And, I don't think it's normal for any other integral values in the data files to have a trailing decimal place.

Reply:
Okay, fair enough. I'll remove them.